### PR TITLE
[FSPE-5231] Update vertical tabs to support title, description and 3rd level

### PR DESCRIPTION
**Pull request by jllr**
https://ctl.atlassian.net/browse/FSPE-5231

@janibolkvadze

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://617884dc46631d071efc81fd7f899f7a35467443-chi-pr.enterprise.ctl.io" target="_blank"> https://617884dc46631d071efc81fd7f899f7a35467443-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1254/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://199ef4616dbf97071a48d6025af23a05e234e658-chi-pr.enterprise.ctl.io" target="_blank"> https://199ef4616dbf97071a48d6025af23a05e234e658-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1261/ for more details.

File: src/chi/components/tabs/tabs.scss [hunk 1]
```diff
@@ -201,6 +201,98 @@
...
+            left: 1rem;
+          }
+        }
+      }
+    }
+
+    &:not(.-sliding) > li.-active:not(.-has-active) > a {
+      background-color: $portal-color-background-table-tr-selected;
+
+      &::after {
```
Comment (mattnickles) [inline src/chi/components/tabs/tabs.scss:282]: Shouldn't this be rendered as an icon button?

Comment (jllr) [reply src/chi/components/tabs/tabs.scss:282]: It is not clear enough in the design. I display it as an pseudo-element when the -active state is present.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://cff210eb0e17d1dc7e69f76c8f270eae057ad771-chi-pr.enterprise.ctl.io" target="_blank"> https://cff210eb0e17d1dc7e69f76c8f270eae057ad771-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1262/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://2ea7282cac8db50b97cacf025d7d263dcf280850-chi-pr.enterprise.ctl.io" target="_blank"> https://2ea7282cac8db50b97cacf025d7d263dcf280850-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1264/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://0e9230da659f58dde96be4f552c47161666be91c-chi-pr.enterprise.ctl.io" target="_blank"> https://0e9230da659f58dde96be4f552c47161666be91c-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1267/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://b8fcced94370d602bac1150145bab46e300f4675-chi-pr.enterprise.ctl.io" target="_blank"> https://b8fcced94370d602bac1150145bab46e300f4675-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1268/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://c0b154a6715138e24b8a5cbc54fbad075cf2131d-chi-pr.enterprise.ctl.io" target="_blank"> https://c0b154a6715138e24b8a5cbc54fbad075cf2131d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1269/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-fpzv3x, https://ops.elasticbox.com/services/instances/i-bcrlst, https://ops.elasticbox.com/services/instances/i-p20whi, https://ops.elasticbox.com/services/instances/i-wmboc0, https://ops.elasticbox.com/services/instances/i-knir6p, https://ops.elasticbox.com/services/instances/i-blyr4k, https://ops.elasticbox.com/services/instances/i-u1cs4l

**Pull request merged**